docs/aurora/performance-tools/aps.md

```diff
@@ -0,0 +1,633 @@
+| review a detailed metric breakdown by memory hierarchy, memory bandwidth
+| information, and correlation by memory objects.
+    Cache Stalls:                            66.60% of Cycles
+| A significant proportion of cycles are spent on data fetches from cache. Use
+| Intel(R) VTune(TM) Profiler Memory Access analysis to see if accesses to L2 or
+| L3 cache are problematic and consider applying the same performance tuning as
+| you would for a cache-missing workload. This may include reducing the data
+| working set size, improving data access locality, blocking or partitioning the
+| working set to fit in the lower cache levels, or exploiting hardware
+| prefetchers.
+    DRAM Stalls:                              3.95% of Cycles
+    DRAM Bandwidth
+       Peak:                                426.75 GB/s
+       Average:                              16.41 GB/s
+       Bound:                                14.05%
+  Vectorization:                             30.22%
+     Instruction Mix:
+       SP FLOPs:                              0.00% of uOps
+       DP FLOPs:                              0.00% of uOps
+       Non-FP:                              200.00% of uOps
+     FP Arith/Mem Rd Instr. Ratio:            0.00
+     FP Arith/Mem Wr Instr. Ratio:            0.00
+ Average PCI Bandwidth:
+   Average PCIe Bandwidth Usage by GPU:
+     Inbound PCIe Read:                     607.84 MB/s
+     Inbound PCIe Write:                   6826.98 MB/s
+   Average PCIe Bandwidth Usage by Network Controller Devices:
+     Inbound PCIe Read:                    6120.50 MB/s
+     Inbound PCIe Write:                   4131.81 MB/s
+ Disk I/O Bound:                              0.94 s             0.11% of Elapsed Time
+      Disk read:                             153.1 KB
+      Disk write:                             50.1 GB
+ Memory Footprint:
+ Resident:
+       Per node:
+           Peak resident set size    :         8944.00 MB (node x4013c2s1b0n0)
+           Average resident set size :         8919.50 MB
+       Per rank:
+           Peak resident set size    :          906.00 MB (rank 24)
+           Average resident set size :          743.29 MB
+ Virtual:
+       Per node:
+           Peak memory consumption    :     64187493.00 MB (node x4013c2s0b0n0)
+           Average memory consumption :     64186991.25 MB
+       Per rank:
+           Peak memory consumption    :      5349075.00 MB (rank 36)
+           Average memory consumption :      5348915.94 MB
+
+Graphical representation of this data is available in the HTML report: /lus/flare/projects/Aurora_deployment/jkwack/JK_AT_Tools/Apps/amr-wind_v3.4.2/build_20250513/jk_test/aps_report_20250519_004709.html
+
+```
+
+
+Download the HTML file and open it on a web browser.
+
+![APS HTML report snapshot](images/aps-01.png "APS HTML report snapshot")  
+[APS HTML report for this example](./results/aps_report_20250519_004709.html)
```

Comment:
might want to move this inline with the previous sentence, so it doesnt render strangely